### DISCUSSION TITLE
Vscode Debug

### DISCUSSION THREAD

**Discussion by ariefsn** [oven-sh/bun#1242] [Q&A] [answered]
hi,
how to debug bun app with vscode?

thanks 🙏🏻

**Comment by Jarred-Sumner** [comment, marked as answer]
We need to do a bunch of stuff before this will work unfortunately.

**Reply by ariefsn** [reply]
thanks @Jarred-Sumner 
will wait.
great work 🥳

**Reply by felipemullen** [reply]
awaiting anxiously 🤗
loving bun so far

**Comment by hungify** [comment]
It's now available 
Just install the extension [Bun for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=oven.bun-vscode)
Then create `lanuch.json`

More: #1106

**Reply by mio-moto** [reply]
This is pretty nifty, but bun currently generates a random prefix (even with `--inspect=localhost:6999/magic-prefix` and just attaching (here: with WSL) needs a change in config every time bun starts. Is there a fix for that? (Or should I open an inssue?)
